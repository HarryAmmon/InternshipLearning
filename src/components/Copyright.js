import React from 'react';

export default class Copyright extends React.Component{
    constructor(props){
        super(props);
    }

    render(){
        return(
            <small>
                Â© {this.props.year} Harry Ammon
            </small>);
    }
}