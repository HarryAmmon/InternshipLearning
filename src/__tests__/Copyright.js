import React from 'react';
import renderer from 'react-test-renderer';
import Copyright from "../components/Copyright.js";

describe("Copyright component",() => {
    it("renderers correctly", () => {
        const tree = renderer
            .create(<Copyright year={2020} />)
            .toJSON();

        console.log(tree);

        expect(tree).toMatchSnapshot();
    })
})