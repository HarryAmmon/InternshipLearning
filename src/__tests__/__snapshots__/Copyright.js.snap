// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright component renderers correctly 1`] = `
<small>
  Copyright Â© 
  2020
   Harry Ammon
</small>
`;
